src/main/java/com/cburch/logisim/gui/start/Startup.java

```diff
@@ -139,41 +139,35 @@ static void doPrint(File file) {
   // Non-used args short keys
   // --------------------------
   //    d    i k         u w t
```

Comment:
Can you update or remove this comment block?

Reply:
Done